src/main/java/org/folio/rest/impl/InvoicesImpl.java

```diff
@@ -1,29 +1,27 @@
 package org.folio.rest.impl;
 
-import static io.vertx.core.Future.succeededFuture;
```

Comment:
Please keep import sections in Eclipse style.

Reply:
imports section reverted